grossmont/computer-science-and-cybersecurity-organization/style.css

```diff
@@ -33,4 +36,14 @@ body {
 body {
   padding-top: 15vh;
   padding-bottom: 15vh;
+}
+
+:root{
+  --primary-color: white;
+  --secondary-color: #171717;
```

Comment:
Nice! no need to remember specific color codes if we assign the colors a constant variable

Reply:
I would suggest you mode this :root to the top of the file, because :root properties are the most important in a website, and it gets loaded first